File: Java/ESAMI/Esame19giugno2017/Python/city_store.py
from negozio import Negozio

class CityStore:

    responsabile: str
    codice_fiscale: str

    def __init__(self, tipo: str, codice: int, indirizzo: str, superficie: int, responsabile: str, codice_fiscale: str) -> None:
        super().__init__(tipo, codice, indirizzo, superficie)
        self.responsabile = responsabile
        self.codice_fiscale = codice_fiscale

    def __str__(self) -> str:
        return "City-Store" + "\t" + super().__str__() + "\t" + self.responsabile + "\t" + self.codice_fiscale + "\t-\t-\t-"

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame19giugno2017/CityStore.java
public class CityStore extends Negozio {

    private String responsabile, codiceFiscale;
    
    public CityStore(String tipo, int codice, String indirizzo, int superficie, String responsabile, String codiceFiscale) {
        super(tipo, codice, indirizzo, superficie);
        this.responsabile = responsabile;
        this.codiceFiscale = codiceFiscale;
    }

    public String getTipo() {
        return tipo;
    }

    public String getNome() {
        return responsabile;
    }

    public String toString() {
        return "City-Store" + "\t" + codice + "\t" + indirizzo + "\t" + superficie + "\t" + responsabile + "\t" + codiceFiscale + "\t-\t-\t-";
    }
    
}
